compiler-rt/lib/rtsan/rtsan_checks.inc

```diff
@@ -0,0 +1,19 @@
+//===-- rtsan_checks.inc ----------------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// List of suppression checks handled by RTSan runtime.
+//
+//===----------------------------------------------------------------------===//
+#ifndef RTSAN_CHECK
+#error "Define RTSAN_CHECK prior to including this file!"
+#endif
+
+// RTSAN_CHECK(Name, SummaryKind)
+// SummaryKind should be a string literal.
+
+RTSAN_CHECK(InCallStack, "in-call-stack")
```

Comment:
Any feedback on this name? See it in use in this .supp below

Reply:
maybe "callstack-contains"? to be that reads nicer `callstack-contains:free` vs `in-call-stack:free`?

Reply:
I like that, @davidtrevelyan WDYT? I think that has a nice grammatical flow to it.

Reply:
I really like callstack-contains 👍